client/src/components/Common/InputFormField/index.js

```diff
@@ -12,22 +12,30 @@ const InputWrapper = styled.div`
   width: 100%;
 `;
 
-const InputField = ({
+const InputFormField = ({
   name,
   value,
   placeholder,
   label,
   changeHandler,
   validityState = {},
   validateInput,
+  onPressEnter,
   children,
   required,
   ...props
 }) => {
+  const handleKeyDown = (e) => {
+    if (e.key === 'Enter' && onPressEnter !== undefined) {
+      onPressEnter();
```

Comment:
Just for fun, another way you could write this would be:

`if(e.key === 'Enter'){
    onPressEnter && onPressEnter(); 
  }
`

Reply:
This is awesome